front/app/containers/Admin/dashboard/users/Charts/GenderChart/index.tsx

```diff
@@ -20,13 +20,15 @@ import { QueryParameters } from './typings';
 
 interface Props extends QueryParameters {
   currentGroupFilterLabel?: string;
+  customFieldId: string;
 }
 
 const GenderChart = ({
   startAt,
   endAt,
   currentGroupFilter,
   currentGroupFilterLabel,
+  customFieldId,
```

Comment:
If we had an easy way to request a single custom field by code from the FE it would be cleaner indeed, but since we don't we'd either have to implement it in the BE or go through the pain of requesting all custom fields and finding the one with the right code... And I think that eventually we want to get rid of these specific charts and instead have a generic user custom field component, ideally reusing the report builder. So I'd rather not spend too much time on it anymore